TWLight/views.py

```diff
@@ -165,3 +174,100 @@ def get_context_data(self, **kwargs):
         context['partner_count'] = Partner.objects.all().count()
         
         return context
+
+class StaffDashboardView(StaffOnly, View):
+
+    def get(self, request, *args, **kwargs):
+        context = {}
+
+        context['accesscode_partners'] = Partner.objects.filter(
+            distribution_type=Partner.CODES)
+
+        return render(request, 'staff.html', context)
+
+    def post(self, request, *args, **kwargs):
```

Comment:
This warrants a second pass for security/safety, since we're running files from the internet through the python interpreter.